autograder/cli/courses/users/enroll.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+import autograder.api.config
+import autograder.api.courses.users.enroll
+import autograder.cli.common
+
+def run(arguments):
+    arguments = vars(arguments)
+
+    arguments['raw-course-users'] = [{
+        'email': arguments['new-email'],
+        'name': arguments['new-name'],
+        'course-role': arguments['new-course-role'],
+        'course-lms-id': arguments['new-lms-id'],
+    }]
+
+    arguments['send-emails'] = not arguments['skip-emails']
+
+    result = autograder.api.courses.users.enroll.send(arguments, exit_on_error = True)
+
+    autograder.cli.common.list_user_op_responses(result['results'], table = arguments['table'])
+    return 0
+
+def main():
+    return run(_get_parser().parse_args())
+
+def _get_parser():
```

Comment:
See if there is code to share with autograder/cli/users/upsert.py.

There may also be some to share in `_get_parser()` with the `-file` variants,
but there is not as many options there.

Note that I am NOT saying that all four of these files have code to share.
I am more thinking that the server user and course user pairs may have code to share with each other.